### PR TITLE
add some `lean{}`, `leanok`s for TU section

### PR DESCRIPTION
**Pull request by edegeltje**
the blueprint not showing progress is such a shame.
i took the liberty to reorder/rephrase some blueprint lemmas to reflect their lean counterparts (lean proves iff instead of implies sometimes)

### CODE REVIEW

File: blueprint/src/code/tu.tex [hunk 1]
```diff
@@ -3,49 +3,104 @@ \section{TU Matrices}
...
+  For the other direction, simply note that $A$ is a submatrix of $C$, and use lemma \ref{lem:code_submatrix_of_tu}.
+\end{proof}
+
+\begin{lemma}[TUness with adjoint identity matrix]
+  \label{lem:code_tu_adjoin_id}
+  \uses{def:code_tu_matrix}
+  \lean{Matrix.TU_adjoin_id_below_iff,Matrix.TU_adjoin_id_above_iff,Matrix.TU_adjoin_id_left_iff,Matrix.TU_adjoin_id_right_iff}
+  \leanok
+  $A$ is TU iff every basis matrix of $\left[ I \mid A \right]$ has determinant $\pm 1$.
+\end{lemma}
```
Comment (edegeltje) [inline blueprint/src/code/tu.tex:92]: technically this lemma doesn't quite agree with the lean statement. if you're ok with it, we can change the blueprint to more accurately reflect what is proven in lean.

Comment (madvorak) [reply blueprint/src/code/tu.tex:92]: I'll leave it up to you.

**Comment by madvorak** [comment]
Thanks!

**Comment by madvorak** [comment]
We probably have a mistake because the lemma is called `Matrix.TU_adjoin_rowUnit_iff` actually.
Can you @edegeltje please check the code and fix the blueprint?

**Pull request merged**